src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoQueryModel.tt

```diff
@@ -309,6 +309,14 @@ namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
 	   {
 			flags.Add("ValidForFlag.NotSqlDw");
        }
+	   if (validForStr.Contains("SqlOnDemand"))
```

Comment:
>   [](start = 1, length = 2)

nit: while you're here could you turn the spacings into tabs? It's inconsistent for some reason in this one